src/qibolab/oneQ_emulator.py

```diff
@@ -0,0 +1,56 @@
+from qibolab.emulator import create_runcard_emulator
+from qibolab.instruments.simulator.models import general_no_coupler_model
+from qibolab.instruments.simulator.pulse_simulator import (
+    PulseSimulator,
+    get_default_simulation_config,
+)
+
+# device parameters: frequency in Hz, time in s
+device_name = "ibmfakebelemQ0"
+sampling_rate = 4500000000.0
+readout_error = [0.01, 0.02]
+lo_freq = 5090167234.445013
+anharmonicity = -336123005.1821652
+drive_freq = 5090167234.445013
+rabi_freq = 125457538.19061986
+T1 = 8.857848970762537e-05
+T2 = 0.00010679794866226273
+nlevel = 3
+
+# simulation parameters
+sim_sampling_boost = 10
+simulate_dissipation = True
+instant_measurement = True
+
+
+def create_oneQ_emulator(runcard_folder: str):
```

Comment:
Similar to the above comment, you can convert this to an actual platform by creating a new directory:
```
onequbit_emulator
    platform.py
```
where `platform.py` is this file with
```suggestion
def create():
```

Reply:
Fixed in e035cb3